test/e2e/nodemaintenance_test.go

```diff
@@ -229,6 +257,7 @@ func nodeMaintenanceTest(t *testing.T, f *framework.Framework, ctx *framework.Te
 		t.Fatal(err)
 	}
 
+	checkOperatorCRD(t)
```

Comment:
Seems like something that could/should be part of a functional test rather than e2e testing

Reply:
can't put it in the unit test in pkg/controller/nodemaintenance/nodemaintenance_controller_test.go ; validation of schema is done by apiserver, whereas that test runs on fake clients. We need a deployed system for the test.